### PR TITLE
Check for repos that have jobs but don't configure trigger

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/assign @cjwagner @fejta 
/cc @BenTheElder @Katharine @pohly

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
```
time="2019-02-27T10:55:11-08:00" level=warning msg="the following repos have jobs configured but do not have the trigger plugin enabled: repo: kubernetes/minikube" component=checkconfig
```

File: prow/cmd/checkconfig/main.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/sirupsen/logrus"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/validation"
-	"k8s.io/test-infra/prow/apis/prowjobs/v1"
+	v1 "k8s.io/test-infra/prow/apis/prowjobs/v1"
```
Comment (cjwagner) [inline prow/cmd/checkconfig/main.go]: 🤔

Comment (stevekuznetsov) [reply prow/cmd/checkconfig/main.go]: Editors, man. WTF?

**Pull request merged**